mec/mec1501/component/espi_vw.h

```diff
@@ -253,11 +257,29 @@
 #define MEC_ESPI_MSVW10_SRC2_POS    14u
 #define MEC_ESPI_MSVW10_SRC3_POS    15u
 
+#define MEC_ESPI_MSVW07_SRC0_VAL    (1U << MEC_ESPI_MSVW07_SRC0_POS)
+#define MEC_ESPI_MSVW07_SRC1_VAL    (1U << MEC_ESPI_MSVW07_SRC1_POS)
+#define MEC_ESPI_MSVW07_SRC2_VAL    (1U << MEC_ESPI_MSVW07_SRC2_POS)
+#define MEC_ESPI_MSVW07_SRC3_VAL    (1U << MEC_ESPI_MSVW07_SRC3_POS)
+#define MEC_ESPI_MSVW08_SRC0_VAL    (1U << MEC_ESPI_MSVW08_SRC0_POS)
+#define MEC_ESPI_MSVW08_SRC1_VAL    (1U << MEC_ESPI_MSVW08_SRC1_POS)
+#define MEC_ESPI_MSVW08_SRC2_VAL    (1U << MEC_ESPI_MSVW08_SRC2_POS)
+#define MEC_ESPI_MSVW08_SRC3_VAL    (1U << MEC_ESPI_MSVW08_SRC3_POS)
+#define MEC_ESPI_MSVW09_SRC0_VAL    (1U << MEC_ESPI_MSVW09_SRC0_POS)
+#define MEC_ESPI_MSVW09_SRC1_VAL    (1U << MEC_ESPI_MSVW09_SRC1_POS)
+#define MEC_ESPI_MSVW09_SRC2_VAL    (1U << MEC_ESPI_MSVW09_SRC2_POS)
+#define MEC_ESPI_MSVW09_SRC3_VAL    (1U << MEC_ESPI_MSVW09_SRC3_POS)
+#define MEC_ESPI_MSVW10_SRC0_VAL    (1U << MEC_ESPI_MSVW10_SRC0_POS)
+#define MEC_ESPI_MSVW10_SRC1_VAL    (1U << MEC_ESPI_MSVW10_SRC1_POS)
+#define MEC_ESPI_MSVW10_SRC2_VAL    (1U << MEC_ESPI_MSVW10_SRC2_POS)
+#define MEC_ESPI_MSVW10_SRC3_VAL    (1U << MEC_ESPI_MSVW10_SRC3_POS)
+
 /*
  * 7 <= v <= 10
  * 0 <= s <= 3
  */
-#define MEC_ESPI_MSVW_07_10_GIRQ_POS(v, s) ((((uint32_t)(v) - 7ul) << 2) + (uint32_t)(s))
+#define MEC_ESPI_MSVW_07_10_GIRQ_POS(v, s) \
+		((((uint32_t)(v) - 7ul) << 2) + (uint32_t)(s))
```

Comment:
What is the 7ul for? Can you drop it?

Reply:
This macro and the similar one for MSVW 0-6 are not used anywhere: removed